x-pack/plugins/synthetics/server/routes/fleet/get_has_zip_url_monitors.ts

```diff
@@ -5,27 +5,18 @@
  * 2.0.
  */
 import { API_URLS } from '../../../common/constants';
-import { ConfigKey } from '../../../common/runtime_types';
 import { SyntheticsRestApiRouteFactory } from '../../legacy_uptime/routes/types';
 
-export const getHasZipUrlMonitorRoute: SyntheticsRestApiRouteFactory = () => ({
+export const getHasIntegrationMonitorsRoute: SyntheticsRestApiRouteFactory = () => ({
   method: 'GET',
-  path: API_URLS.SYNTHETICS_HAS_ZIP_URL_MONITORS,
+  path: API_URLS.SYNTHETICS_HAS_INTEGRATION_MONITORS,
   validate: {},
   handler: async ({ savedObjectsClient, server }): Promise<any> => {
     const monitors = await server.fleet.packagePolicyService.list(savedObjectsClient, {
       kuery: 'ingest-package-policies.package.name:synthetics',
```

Comment:
```suggestion
      kuery: 'ingest-package-policies.package.name:synthetics and not ingest-package-policies.package.is_managed:true',
```
we need to filter unmanaged policies. filter will be like that ^^ though i am not sure, you may have to play around to find the filter.